scenariolib/event_pageview.go

```diff
@@ -68,6 +71,22 @@ func (ve *ViewEvent) Execute(v *Visit) error {
 		return nil
 	}
 
+	ve.computeClickRank(v). // Randomize a click rank if the clickRank is -1
```

Comment:
Kind of a micro-optimization, but just putting the thought here:

You would not need to compute anything if `willExecute` is false.